extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -415,11 +425,20 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 		await this._azureRecommendationNotAvailableText.updateCssStyles({ 'display': 'none' });
 		await this._recommendedConfigurationText.updateCssStyles({ 'display': 'block' });
 		await this._vmRecommendedConfigurationText.updateCssStyles({ 'display': 'block' });
+		await this._viewDetailsLink.updateCssStyles({ 'display': 'block' });
 
 		this._recommendedConfigurationText.value = skuRecommendation;
 		this._vmRecommendedConfigurationText.value = vmRecommendation;
 	}
 
+	public async loadingSKURecommendation() {
```

Comment:
let's make the access modifier private since called only in same class.

Reply:
It is called outside in skuRecommendationPage. So, it has to be public only.